services/web/server/src/simcore_service_webserver/computation_subscribe.py

```diff
@@ -114,6 +114,7 @@ async def events_message_parser(app: web.Application, data: bytes) -> None:
 
 
 APP_RABBITMQ_POOL_KEY = f"{__name__}.pool"
+_RABBITMQ_INTERVAL_BEFORE_RESTARTING_CONSUMER_S = 2
```

Comment:
MINOR: add "WAIT_BEFORE" in the name analogous to tenacity naming

Reply:
Can we conclude some best practices from here? e.g.

- Every restart should include a timeout
- This timeout can be fix or random, ... (as tenacity options) depending on the context
and

- Should we create an generic/abstract mechanism to enforce this?
- Should we review other parts of the code and apply consistently

Reply:
we should use tenacity

Reply:
with retry_unless_exception_type(CancelledError) | retry_unless_value(continue==False) and drop all this custom made code